### PR TITLE
External Home Page

### PR DESCRIPTION
**Pull request by dzwood**
## What's this for? 
We want to add an external landing page for the home page when a user is not logged in.

### NOTE: a solution for the background video is still being worked out, this is just for testing purposes.
* added Text Carousel package: https://www.npmjs.com/package/react-simple-typewriter

## Demo
* Go to home page.

### CODE REVIEW

File: components/DiscoverFilterSection/DiscoverFilterSection.js [hunk 1]
```diff
@@ -14,6 +14,8 @@ const DiscoverFilterSection = ({ contentId, title }) => {
     variables: { id: contentId, first: 3 },
   });
 
+  console.log({ contentId });
```
Comment (asleepingpanda) [inline components/DiscoverFilterSection/DiscoverFilterSection.js:17]: Don't forget to remove this.

**Review by asleepingpanda — CHANGES_REQUESTED** [review]

File: components/DiscoverFilterSection/DiscoverFilterSection.js (hunk 1 above)
Comment (dzwood) [reply components/DiscoverFilterSection/DiscoverFilterSection.js:17]: Removed :)

**Comment by asleepingpanda** [comment]
@dzwood could you update the navigation bar color to use the navy as the background instead of gray?

**Comment by asleepingpanda** [comment]
Could we get the "Contact Us" buttons to open in new tabs, please

**Comment by dzwood** [comment]
> @dzwood Just caught something testing. The main "I Already Attend" button looks awesome when you hover, and I would love that same `white` background to be applied to the "Watch Online" button
> Screen Shot 2021-08-04 at 9 07 16 AM

Good Catch!

**Review by asleepingpanda — APPROVED** [review]

**Pull request merged**